ddtrace/contrib/aiohttp/middlewares.py

```diff
@@ -126,6 +126,18 @@ async def on_prepare(request, response):
     request_span.finish()
 
 
+async def on_prepare(request, response):
```

Comment:
Do we know when this signal is sent? Is the server done responding? Is there anything after this signal is sent that we would want to capture with the request span? Can we do with just the callback?

Reply:
Maybe. My approach with this change is to avoid breaking any existing behavior. Do you think it's important to remove this signal handler if possible?

Reply:
Not necessary. I had missed the fact that the signal was already being handled and that this change was ensuring streaming responses were closed. 

What happens right now with streaming responses?

Reply:
They have incorrect durations as described here https://github.com/DataDog/dd-trace-py/issues/1619